server/util/fileresolver/fileresolver.go

```diff
@@ -29,7 +29,11 @@ func (r *fileResolver) Open(name string) (fs.File, error) {
 		}
 	}
 
-	runfilePath, err := runfiles.Rlocation(name)
+	prefix := "_main"
+	if strings.HasPrefix(name, "/") {
```

Comment:
Is this meant to be

```suggestion
	if !strings.HasPrefix(name, "/") {
```
so that there is only a single slash? How about cleaning the path instead so that names starting with `../` are supported (which correspond to files in external repos)?